datafusion/core/tests/sql/parquet.rs

```diff
@@ -309,11 +309,11 @@ async fn parquet_query_with_max_min() {
     let sql = "SELECT max(c1) FROM foo";
     let actual = execute_to_batches(&ctx, sql).await;
     let expected = vec![
-        "+-------------+",
-        "| MAX(foo.c1) |",
-        "+-------------+",
-        "| 3           |",
-        "+-------------+",
+        "+---------+",
+        "| MAX(c1) |",
```

Comment:
I am not sure what is responsible for removing the relation name `foo` but I think it is a good change (and more consistent with postgres)